sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -67,9 +65,13 @@ int coap_parse(coap_pkt_t *pkt, uint8_t *buf, size_t len)
         pkt->token = NULL;
     }
 
+    coap_optpos_t *optpos = pkt->options;
+    unsigned option_count = 0;
```

Comment:
Is using just `unsigned` instead of the "more standard" `unsigned int` according to the used style guide?

Reply:
In RIOT just ```unsigned``` seems to be "more standard". Personally I prefer less typing and one-word type identifiers...